tests/test_password_input.py

```diff
@@ -0,0 +1,61 @@
+from vorta.views.partials.password_input import PasswordInput, PasswordLineEdit
+
+
+def test_create_password_line_edit(qtbot):
+    password_line_edit = PasswordLineEdit()
+    qtbot.addWidget(password_line_edit)
+    assert password_line_edit is not None
+
+
+def test_password_line_get_password(qtbot):
+    password_line_edit = PasswordLineEdit()
+    qtbot.addWidget(password_line_edit)
+
+    password_line_edit.setText("test")
+    assert password_line_edit.get_password() == "test"
+
+
+def test_password_line_visible(qtbot):
+    password_line_edit = PasswordLineEdit()
+    qtbot.addWidget(password_line_edit)
+    assert password_line_edit.visible is False
+
+    password_line_edit.toggle_visibility()
+    assert password_line_edit.visible
```

Comment:
Why do you use `is` inconsistently here?

Reply:
There were some pep warning regarding use of `x == True:` instead of `x:`. 

I've changed the code to
```
    assert not password_line_edit.visible

    password_line_edit.toggle_visibility()
    assert password_line_edit.visible
```